Blockzilla/SettingsViewController.swift

```diff
@@ -88,46 +88,33 @@ class SettingsTableViewSearchCell: SettingsTableViewCell {
 
 class SettingsViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
     enum Section: String {
-        case search, integration, privacy, security, performance, mozilla
+        case search, integration, privacy, mozilla
         
         var numberOfRows: Int {
             switch self {
             case .search: return 2
             case .integration: return 1
-            case .privacy: return 4
-            case .security: return 1
-            case .performance: return 1
-            case .mozilla: return 3
+            case .privacy:
+                if BiometryType(context: LAContext()).hasBiometry { return 4 }
+                return 3
+            case .mozilla: return 2
             }
         }
         
-        var headerText: String {
+        var headerText: String? {
             switch self {
             case .search: return UIConstants.strings.settingsSearchTitle
-            case .integration: return UIConstants.strings.toggleSectionIntegration
             case .privacy: return UIConstants.strings.toggleSectionPrivacy
-            case .security: return UIConstants.strings.toggleSectionSecurity
-            case .performance: return UIConstants.strings.toggleSectionPerformance
             case .mozilla: return UIConstants.strings.toggleSectionMozilla
+            default: return nil
             }
         }
-        
-        var hasFooter: Bool {
-            switch self {
-            case .privacy, .mozilla: return true
-            case .search, .integration, .security, .performance: return false
-            }
-        }
-        
+
         static func getSections(deviceHasBiometrics: Bool) -> [Section] {
-            var sections: [Section] = [.search, integration, privacy, .performance, .mozilla]
-            if deviceHasBiometrics { sections.insert(.security, at: 3) }
```

Comment:
If we're not going to use this deviceHasBiometrics value anymore, we should change the function definition.